README.rst

```diff
@@ -714,7 +714,7 @@ Code example:
     An ``ImproperlyConfigured`` exception is raised if group already has an
     item registered at ``item_position``.
 
-    Registeration of a model link or a generic link is only possible.
+    It is only possible to register links to specific models or custom URL.
```

Comment:
"generic link" => "custom URL" should be changed everywhere, not only here.

Reply:
right, thanks!